common/src/main/java/io/netty/util/concurrent/CompleteFuture.java

```diff
@@ -140,6 +140,9 @@ public boolean isCancelled() {
         return false;
     }
 
+    /**
+     * @param mayInterruptIfRunning this value has no effect in this implementation.
```

Comment:
at a minimum we need {@inheritDoc}, and in general I'm not convinced this change is necessary as the `cancel` operation itself is indicated as best effort in the original javadocs and therefore the parameter may not be used in general.

Reply:
if a param value does not used in the method, then it should be indicated by the javadoc, but the original javadoc from `java.util.concurrent.Future#cancel` does not clearly indicate it, so the JVM team itself indicate it when the implementation method does not use the param `mayInterruptIfRunning`, please refer the javadoc of `java.util.concurrent.CompletableFuture#cancel` and `java.util.concurrent.ForkJoinTask#cancel` ;
 which javadocs clearly indicate that `mayInterruptIfRunning` has no effect in the implementation.